src/components/FilterPanel/FilterPanel.hook.ts

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react';
+
+export const useFilterPanel = (initialValue: string) => {
```

Comment:
useFilterPanel currently accepts initialValue: string and returns { selected, handleSelect } with no explicit types. The hook is also unused in the rest of the diff. The hook could be more flexible and better-typed (and should be exported with a stable signature).

**Sugerencia:**
```typescript
Make the hook generic or typed and memoize the handler. Example:

export const useFilterPanel = (initialValue?: string) => {
  const [selected, setSelected] = useState<string | undefined>(initialValue);
  const handleSelect = useCallback((value: string) => setSelected(value), []);
  return { selected, handleSelect } as const;
};

Also remove the hook if not used.
```